pixi/src/ui/report/BooleanCheckReportView.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default class BooleanCheckReportView {
+  constructor(doc, id) {
+    this.container = doc.getElementById(id);
+    this.label = this.container.querySelector('label');
+  }
+
+  render(report) {
+    if (report === true) {
+      this.label.textContent = 'Passed';
+      this.container.classList.add('passed');
+    } else if (report === false) {
+      this.label.textContent = 'Failed';
+      this.container.classList.add('failed');
+    } else {
+      this.label.textContent = 'Check manually';
```

Comment:
move this into a separate View implementation as suggested in one of my previous comments.